src/v2/Apps/Auctions/Routes/CurrentAuctions.tsx

```diff
@@ -29,7 +32,13 @@ export const CurrentAuctionsFragmentContainer = createFragmentContainer(
       fragment CurrentAuctions_currentAuctions on SaleConnection {
         edges {
           node {
-            id
+            slug
+            name
+            href
+            startAt
+            endAt
+            liveStartAt
```

Comment:
We can probably remove these unused props as we're capturing everything in the fragment spread below.

Reply:
Removed.